libs/SmartStore/SmartStore/Classes/SFStoreCursor.h

```diff
@@ -82,13 +82,13 @@ NS_SWIFT_NAME(StoreCursor)
  * Run query and resturn JSON serialized representation of the cursor.
  * @return JSON serialized representation of this object.
  */
-- (NSString*)getDataSerialized:(SFSmartStore*)store error:(NSError**)error;
+- (nullable NSString*)getDataSerialized:(SFSmartStore*)store error:(NSError**)error;
```

Comment:
Both of these methods have cases that return nil

Reply:
@bbirman Isn't this technically a breaking change?  It's a good change because it forces the dev to handle this case which could avoid a crash but still.

Reply:
I thought we've changed nullability before in .x releases but I'm good either way

Reply:
Leave it in. It's the correct value now anyway.

Reply:
Agreed, just a gentle reminder for the future.